DiamondLightSource/python-artemis
# Add deadtime optimisation

Fixes #754 

Link to dodal PR (if required): https://github.com/DiamondLightSource/dodal/pull/102

### To test:
1. Confirm deadtime optimisation logic is the same of that in the GDA script mx-config/scripts/attenuation_optimiser.py, in the run_xspress3mini_optimisation_routine()

2. Confirm new tests run and are sensible
